### PR TITLE
Create SECURITY.md

### LINKED ISSUE #4447: Create a Security Policy
### Should this be an RFC?

- [X] This is not a substantial change

### Which package is this a feature request for?

Lit Core (lit / lit-html / lit-element / reactive-element)

### Description

Hi again, would you consider creating a SECURITY.md file to disclosure how users should report a vulnerability found?

A Security Policy is a GitHub standard document (SECURITY.md) that can be found in the "Security Tab" to instruct users how to report vulnerabilities in a safe and efficient way.

It appears in both the about page and the security dashboard (see images in Additional Context)

It is a [Scorecard Recommendation](https://github.com/ossf/scorecard/blob/main/docs/checks.md#security-policy) (being a security measure of medium priority) and a [Github Recommendation](https://docs.github.com/en/code-security/getting-started/securing-your-repository#setting-a-security-policy).

I'll be submitting a simple model for Security Policy considering the github vulnerability report feature that is already enabled. Please feel free to suggest a different one or request as many changes as you think it needs.

Here are some examples of Security Policies:

- https://github.com/tensorflow/tensorflow/security
- https://github.com/JakeChampion/fetch/security

## Additional Context 

image

### Alternatives and Workarounds

None

### PR DESCRIPTION
**Pull request by joycebrum**
Closes #4447 

Hi, I've followed the simplest model I could think, let me know how to make it a better fit for lit.

Another possibility, since lit is a google project, would be listing the bug bounty project as a possibility for vulnerability report.

Thanks!

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are applied only to the latest release.
```
Comment (augustjk) [inline SECURITY.md:5]: I think we should consider applying security updates to at least the previous major release.

**Review by augustjk — COMMENTED** [review]
Thank you for the PR!
Added additional reviewers for comments.

File: SECURITY.md (hunk 1 above)
Comment (joycebrum) [reply SECURITY.md]: I've updated it to the latest two major versions. See if it is good or if it should be rephrased.

**Review by rictic — APPROVED** [review]
This is great to have documented clearly

**Review by augustjk — APPROVED** [review]
Thank you!

**Pull request merged**